docs/cache/manage/limits.md

```diff
@@ -27,61 +27,63 @@ The limits on this page are soft limits that can be altered unless specifically
-| ListFetch                | Number of elements in response, or 1 if list is not found       |
-| ListConcatenateBack      | Number of elements in request                                   |
-| ListConcatenateFront     | Number of elements in request                                   |
-| ListLength               | 1                                                               |
-| ListPopBack              | 1                                                               |
-| ListPopFront             | 1                                                               |
-| ListPushBack             | 1                                                               |
-| ListPushFront            | 1                                                               |
-| ListRemoveValue          | 1                                                               |
-| ListRetain               | 1                                                               |
-| SetAddElement            | 1                                                               |
-| SetAddElements           | Number of elements in request                                   |
-| SetFetch                 | Number of elements in response, or 1 if set is not found        |
-| SetRemoveElement         | 1                                                               |
-| SetRemoveElements        | Number of elements in request                                   |
-| SetContainsElement       | 1                                                               |
-| SetContainsElements      | Number of elements in request                                   |
-| SetLength                | 1                                                               |
-| SortedSetPutElement      | 1                                                               |
-| SortedSetPutElements     | Number of elements in request                                   |
-| SortedSetFetchByRank     | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetFetchByScore    | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetGetScore        | 1                                                               |
-| SortedSetGetScores       | Number of elements in request                                   |
-| SortedSetRemoveElement   | 1                                                               |
-| SortedSetRemoveElements  | Number of elements in request                                   |
-| SortedSetGetRank         | 1                                                               |
-| SortedSetIncrementScore  | 1                                                               |
-| SortedSetLength          | 1                                                               |
-| SortedSetLengthByScore   | 1                                                               |
+| API Name                 | Batch API | Operations                                                        |
```

Comment:
Do we want to call this `Batch API`? At one point we were calling these "multi element APIs" rather than "batch", so should this be like `Multi Element`?

Reply:
good question. If we add `multiget` and `multiset` soon, I would think of those as "batch APIs". It's a little murkier for me whether stuff inside of a collection would also qualify as "batch". I would probably vote for multi-element but I don't feel strongly, maybe @allenheltondev and/or @danielamiao can weigh in.

Reply:
Hm, in the context of throttling, I think `multiget` and `multiset` would behave the same way as "multi element" APIs, so maybe in this context it doesn't matter? Like, if we added those APIs I think they'd just be in this same table rather than a different table?

Reply:
yes, i agree. so i think the table and the docs will only need to say one thing, either way. just a matter of deciding which one of the two things to say, in case the word "batch" has other connotations/meanings to some people like it kinda does for me.

but i don't feel strongly about it.

Reply:
I also prefer multi-element because something like `DictionaryFetch` doesn't intuitively sound like a "batched" API to me.

Reply:
Renamed to multi-element in [fe61606](https://github.com/momentohq/public-dev-docs/pull/602/commits/fe61606206dd47c9f27c4197f2eaac13b5bba999)